citi-onboarding/DADe-onepage
# feat(feature/como-trabalhamos): HowWeWork view

## What I did

- Create How We Work view for both desktop and mobile versions
- Add How We Work Id to Navbar (navbar still appears on top of the section)

| Mockup | Real |
|--------|------|
|![Captura de Tela_select-area_20210624114546](https://user-images.githubusercontent.com/61971582/123284239-7aa0fb80-d4e2-11eb-961c-483f81421111.png)|![real_trabalhamos_topo](https://user-images.githubusercontent.com/61971582/123284271-81c80980-d4e2-11eb-9a60-dbbc3e9aac24.png) ![real_trabalhamos_baixo](https://user-images.githubusercontent.com/61971582/123284294-868cbd80-d4e2-11eb-83d2-0ce6e7dcfd51.png)

| Mockup | Real |
|--------|------|
|![Captura de Tela_select-area_20210624114600](https://user-images.githubusercontent.com/61971582/123284431-a0c69b80-d4e2-11eb-837e-ecb01194e96d.png)|![real_mobile](https://user-images.githubusercontent.com/61971582/123284452-a45a2280-d4e2-11eb-98dc-7ad29718a68a.png) ![real_mobile2](https://user-images.githubusercontent.com/61971582/123284476-a91ed680-d4e2-11eb-971e-e3882641394d.png)

## How to test

1. Open the terminal and go to the folder of this project

2. In terminal:
   1. `git checkout develop`
   2. `git pull`
   3. `git checkout feature/como-trabalhamos`
   4. `git pull origin feature/como-trabalhamos`
   5. `npm install`
   6. `cd server/`
   7. `node index.js`
   8. Open a new terminal
   9. `cd client/`
   10. `npm start`

3. In your browser go to http://localhost:3000 (should be open already)

4. Check if the component looks just like the mockup in different screen sizes, and both the mobile and desktop versions.

5. Check if you can click the 'Como Trabalhamos' button in the navbar (desktop and mobile) and see if it takes you to the correct section (navbar still appears on top of section)

6. Read the changed files section on GitHub 

7. Test the things that you think are worth testing, even the ones that are not in this description
